utils/ping-editorial-team.js

```diff
@@ -21,12 +21,13 @@ WARNING: Duplicate Posts Found
 -----------------------------------------------
 ${msg}
 `);
+  console.error(msg);
```

Comment:
```suggestion
```
Hey, Kris, this isn't a proper review, but I was skimming over it and noticed this!

Reply:
Thanks for catching this! Totally forgot about this line.